homeassistant/components/apple_tv/strings.json

```diff
@@ -0,0 +1,67 @@
+        "data": {
+          "device_input": "Device"
+        }
+      },
+      "reconfigure": {
+        "title": "Device reconfiguration",
+        "description": "This Apple TV is experiencing some connection difficulties and must be reconfigured."
+      },
+      "pair_with_pin": {
+        "title": "Pairing",
+        "description": "Pairing is required for the `{protocol}` protocol. Please enter the PIN code displayed on screen. Leading zeros shall be omitted, i.e. enter 123 if the displayed code is 0123.",
+        "data": {
+          "pin": "PIN Code"
+        }
+      },
+      "pair_no_pin": {
+        "title": "Pairing",
+        "description": "Pairing is required for the `{protocol}` service. Please enter PIN {pin} on your Apple TV to continue."
+      },
+      "service_problem": {
+        "title": "Failed to add service",
+        "description": "A problem occurred while pairing protocol `{protocol}`. It will be ignored."
+      },
+      "confirm": {
+        "title": "Confirm adding Apple TV",
+        "description": "You are about to add the Apple TV named `{name}` to Home Assistant.\n\n**To complete the process, you may have to enter multiple PIN codes.**\n\nPlease note that you will *not* be able to power off your Apple TV with this integration. Only the media player in Home Assistant will turn off!"
+      }
+    },
+    "error": {
+      "device_not_found": "Device could not be found on network, please try again.",
+      "device_already_configured": "This device has already been configured. Please pick another device.",
+      "no_usable_service": "A device was found but could not identify any way to establish a connection to it. If you keep seeing this message, try specifying its IP address or restarting your Apple TV.",
+      "unknown": "Unexpected error! Check log file for further details.",
+      "auth": "Authentication error (you might have entered an invalid PIN code)."
+    },
+    "abort": {
+      "device_not_found": "Device could not be found on network, please try again.",
+      "already_configured": "Device is already configured!",
+      "updated_configuration": "Updated existing configuration!",
+      "unrecoverable_error": "An unrecoverable error occurred! Check log file for further details.",
+      "device_did_not_pair": "No attempt to finish pairing process was made from the device.",
+      "backoff": "Device does not accept pairing reqests at this time (you might have entered an invalid PIN code too many times), try again later.",
+      "invalid_config": "The configuration for this device is incomplete. Please try adding it again.",
+      "auth": "Authentication error. You might have entered an invalid PIN code or not have permission to pair via this protocol. Please check access settings on your Apple TV.",
+      "already_in_progress": "Config flow for device is already in progress.",
```

Comment:
Not sure if we need to add this key (`already_in_progress`), which is common for all flows

Reply:
If the flow can be aborted with that reason we need a string for it. We can use a common reference for the value though.

Reply:
I believe this string is wrong and I can probably remove if it is included anyways.

Reply:
It can happen when setting the config flow unique id, eg in a discovery flow. Keep the string but use the common reference.